app/javascript/components/Pages/AdoptionEvening/index.jsx

```diff
@@ -19,14 +18,16 @@ export default ({
   cta_text,
   cta_button,
   body,
-  events
+  events,
+  hero_image,
+  hero_alt
 }) =>
   <Layout withHeader withFooter>
     <HeroWithImage
       headline={title}
       deck={strapline}
-      image={heroimage}
-      alt="Mother holding a baby showing information on a tablet to another young child"
+      image={hero_image}
+      alt={hero_alt || "Mother holding a baby showing information on a tablet to another young child"}
```

Comment:
This default alt text is very specific 😄 Can we just leave it blank if it's not provided?